src/jshint.js

```diff
@@ -3561,6 +3542,25 @@ var JSHINT = (function() {
         }
       }
 
+      // absorb the assignment before processing the label so that use of the label
```

Comment:
It seems  like this comment (here and below) is now inaccurate--it's describing the behavior that this change removes. Is that right? If so, with your permission, I'll remove it from the patch when I merge.